scoper.inc.php

```diff
@@ -88,6 +88,11 @@
 			->name( '#^autoload.php$#' )
 			->depth( '== 0' )
 			->in( 'vendor/google/apiclient-services' ),
+
+		Finder::create()
```

Comment:
Let's add a comment here to describe what this is for. For example:

```suggestion
        // Temporary support for `GoogleAnalyticsAdminV1alphaAdSenseLink` as it doesn't exist in the API client yet.
		Finder::create()
```